sftp.c

```diff
@@ -934,7 +971,14 @@ do_globbed_ls(struct sftp_conn *conn, const char *path,
 			mprintf("%s\n", lname);
 			free(lname);
 		} else {
-			mprintf("%-*s", colspace, fname);
+			/* *** ICL Modification *** if file has .iron extension, prefix with lock icon */
+			int offset = iron_extension_offset(fname);
+			if (offset > 0) {
+				mprintf("%s%-*s", ICL_LOCK_ICON, (colspace - 2), fname);
+			} else {
+				mprintf("  %-*s", (colspace - 2), fname);
+			}
+			/* *** */
 			if (c >= columns) {
```

Comment:
Decided not to. One prints to a string, one needs to peel out the dirname of the file then add the icon, etc. They are different enough it's not worth it.